armi/runLog.py

```diff
@@ -503,9 +515,41 @@ class RunLogger(logging.Logger):
         :id: I_ARMI_LOG
         :implements: R_ARMI_LOG
 
+        This logger makes it easy for users to add log statements to
+        and ARMI application, and ARMI will control the flow of
+        those log statements. In particular, ARMI overrides the normal
+        Python logging tooling, to allow developers to pipe their log
+        statements to both screen and file. This works for stdout and
+        stderr.
+
+        The log statements themselves can be any text a user wants to
+        record during a run. For instance, basic notifications of what
+        is happening in the run, or basic warnings about something that
+        could be an issue, so you want people to know about. Likewise,
+        full errors can be recorded with this tool. These different
+        types of log statements are easily identified by specifying the
+        "verbosity" of the log statement, in the code where you do the
+        logging. The most important verbosities are: error, warning,
+        info (which is just to add information to the log), and debug
+        (which is meant to be extremely detailed and not always
+        desired).
+
+        And at the end of the ARMI-based simulation, the analyst doing
+        the simulation will have a full record of potentially intersting
+        information they can use to understand their run.
+
     .. impl:: Logging is done to the screen and to file.
         :id: I_ARMI_LOG_IO
         :implements: R_ARMI_LOG_IO
+
+        By default, this logger prints all log messages during an ARMI
+        simulation to screen and to a plain text file. That is, at any
```

Comment:
Can you explain this a little better? I see if rank is 0 the `StreamHandler` is used and otherwise the `FileHandler` is used. -- I THINK that's the ticket but am not sure.

Reply:
When you use the `runLog()`:

```python
runLog.info("whatever")
```

This gets printed to the screen AND recorded to the log file. 

That is what I was hoping to explain.

Reply:
I understand the outcome but I don't see how that is implemented in the code. All I see is what I mentioned above, which doesn't align with what's in the description. (I trust the description is correct, I just need help finding that in the code)